lib/project_types/script/layers/application/project_dependencies.rb

```diff
@@ -9,13 +9,15 @@ def self.bootstrap(ctx:, language:, extension_point:, script_name:)
 
         def self.install(ctx:, language:, extension_point:, script_name:)
           dep_manager = Infrastructure::DependencyManager.for(ctx, language, extension_point, script_name)
-
           unless CLI::UI::Frame.open(ctx.message('script.project_deps.checking_with_npm')) do
```

Comment:
I wish it was more obvious that this block needs to return a truthy value, but I guess theres not much we can do or even test for

Reply:
I guess if we read the documentation: https://github.com/Shopify/shopify-app-cli/blob/a0bcefbfa5baaad2857f8f9ff1fda20472a78393/vendor/deps/cli-ui/lib/cli/ui/frame.rb#L18.  It's more clear.

Reply:
Yeah. It just may not be super clear to devs who haven't looked into it